packages/server/yarn.lock

```diff
@@ -995,30 +995,6 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
-"@budibase/backend-core@^1.0.97":
```

Comment:
Annoying how the lerna dependencies keep getting re-added then removed. Chatted with cheeks about this before, looks like they come back in when a `yarn install` is done inside one of the packages. 

Trying to spread awareness that we should always run a `yarn bootstrap` to manage any installation of packages which will prevent the budibase dependencies being added to the lock file. We could maybe add something to CI or a pre-commit hook to check if a `.lock` file contains another dependency from the mono repo and fail just so we don't keep seeing these diffs

Reply:
ah interesting, I didn't realise that, what happened here was I installed a new version of `pouchdb` - right now I don't think we have a way to do this through lerna easily, perhaps we need to provide some tooling to add new packages like `yarn add pouchdb`

Reply:
you're right doing a yarn add will bring them back too. Normally I run a `yarn clean && yarn bootstrap` to re-sanitise the lock files after that

Reply:
I will actually add a top level operation which runs through all of the repos and runs a yarn clean, then runs a bootstrap to update.